### PR TITLE
chore(lint): remove global declarations from JSHint

### PR DESCRIPTION
**Pull request by TDA**
Follow up from #2597 
Removed "global" declarations in tests directory.
@pdehaan 
@zaach 
@vladikoff

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,6 +1,8 @@
 {
   "extends": "fxa/client",
-
+  "env": {
+  	"jquery": true
+  },
```
Comment (pdehaan) [inline .eslintrc]: Why add jQuery here?

Comment (TDA) [inline .eslintrc]: The '$' global that i removed in one of the files needs this.

**Comment by TDA** [comment]
@pdehaan Travis failed because we STIIIIILLL have jshint in the repo, we need to get that merged, and then this needs to be rebased.

File: tests/functional/lib/helpers.js [hunk 2]
```diff
@@ -117,7 +116,6 @@ define([
       .end()
 
       .execute(function () {
-        /* global $ */
```
Comment (TDA) [inline tests/functional/lib/helpers.js]: @pdehaan This file, to be precise.

Comment (pdehaan) [inline tests/functional/lib/helpers.js]: Maybe try `//*eslint-env jquery */` here instead of adding it globally in .eslintrc?

**Comment by pdehaan** [comment]
@TDA: Can you rebase pls? I merged your other PR and looks like we may have some conflicts.

**Comment by TDA** [comment]
@pdehaan Doing :) Thanks for merging the other one!

File: app/scripts/head/start.js [hunk 3]
```diff
@@ -1,7 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-/*globals FxaHead */
+ /*globals FxaHead */
```
Comment (pdehaan) [inline app/scripts/head/start.js]: Nit: remove leading space here.

**Comment by pdehaan** [comment]
Apart from the one nit, I like this. Good work @TDA!

File: app/scripts/head/start.js (hunk 3 above)
Comment (TDA) [inline app/scripts/head/start.js]: oops, fixed :) @pdehaan

**Pull request merged**